conf.py

```diff
@@ -73,9 +73,9 @@
             "https://github.com/GenericMappingTools",
         ),
         (
-            '<i class="fab fa-gitter fa-lg"></i>',
-            "Gitter chat",
-            "https://gitter.im/GenericMappingTools/Lobby",
+            '<i class="fab fa-discourse fa-lg"></i>',
```

Comment:
It's better to open these links in new tabs.

Reply:
:+1: agreed

Reply:
Done